packages/lambda-xyz/src/routes/tile.ts

```diff
@@ -138,7 +140,9 @@ export async function Wmts(req: LambdaContext, wmtsData: TileDataWmts): Promise<
     // null
     const tileSet = await loadTileSet(req, wmtsData.name, wmtsData.projection ?? EPSG.Google);
 
-    const xml = tileSet == null ? null : buildWmtsCapability(host, req, tileSet);
+    const provider = await Aws.tileMetadata.Provider.get(TileMetadataTag.Production);
```

Comment:
I think we will likely need more than one provider in the future, how about adding a name to this?

maybe just hard code it to `linz` for now.

Reply:
How about `"default"` (or `main`) rather than `linz` to be generic.

Reply:
I think we should hold off supporting more than one provider until such as situation arises rather than future proofing now.

Reply:
The problem we will have is since there is no provision for more than one it would require a data migration to fix it. I am not a huge fan of migrating data in production environments if i can avoid it :)  hard coding something as the name means we dont have to migrate data in the future. I could see us serving other imagery sets that we have access to that may not be `linz` as the provider.

looking at the code the provider name was ` 'Land Information New Zealand'` so what is wrong with `linz` as being the provider name?

Reply:
If you are strongly in favour of adding name I will but I'm not happy with linz as the value to lookup. `main` or `default` make much more sense to me.

> looking at the code the provider name was 'Land Information New Zealand' so what is wrong with linz as being the provider name?

Because linz is nz specific and the main point of this PR is to remove linz specific code.

Reply:
Another option is to add provider to the tileset record and use that as the name to lookup

Reply:
> Another option is to add provider to the tileset record and use that as the name to lookup

Sounds good to me, lets just do that.

Reply:
Of course that also requires a migration so I would suggest holding off until we need it

Reply:
Now stores and retrieves record as `pv_main_*`